content/docs/04-clusters/02-data-center/01-maas/05-install-manage-maas-pcg.md

```diff
@@ -0,0 +1,401 @@
+---
+title: "Install and Manage MAAS Gateway"
+metaTitle: "Install and Manage MAAS Private Cloud Gateway"
+metaDescription: "Learn how to install and manage the MAAS Private Cloud Gateway in Palette."
+hideToC: false
+fullWidth: false
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+import PointsOfInterest from 'shared/components/common/PointOfInterest';
+
+# Overview 
+
+The Private Cloud Gateway (PCG) enables support for private cloud or data center environments. You can set up the cloud gateway as a single-node or three-node cluster based on your requirements for high availability (HA).  
+<br />
+
+<InfoBox>
+
+For production environments, we recommend using three nodes. If you initially set up the gateway with one node, you can resize it at a later time. 
+
+</InfoBox>
+
+
+Palette provides an installer in the form of a Docker container that is temporarily deployed on your laptop, workstation, or jump box. You can use the installer on any linux x86-64 system that has a Docker daemon installed and connectivity to Palette and the MAAS identity endpoint. 
+
+<br />
+
+
+<WarningBox>
+
+The installer does not currently work on MacOS running on Apple Silicon.
+
+</WarningBox>
+
+
+## Prerequisites
+
+- Canonical [MAAS installed](https://maas.io/docs/how-to-install-maas), set up, and available in your environment.
+
+
+- A linux computer with a Docker daemon installed and a connection to Palette and the MAAS endpoint. The installer must be invoked on a linux system. We have tested the gateway installation using Ubuntu 20.04 on x86-64.
+
+
+- Private cloud gateway IP requirements: <br /><br /> 
+    
+    - For a single-node gateway, one IP address available in the MaaS subnet for the PCG, or three available IP addresses for a three-node gateway.
+    <br />
+
+    - One IP address available in the MaaS subnet for the Kubernetes api-server endpoint, when deploying a three-node gateway.
+
+
+- Sufficient available IPs within the configured MaaS subnets.
+
+<WarningBox>
+
+By default, the MAAS Kubernetes pack uses a pod classless inter-domain routing (CIDR) range of 192.168.0.0/16. Ensure that the pod CIDR range for any clusters you deploy after setting up the PCG do not overlap with the network used by the bare metal machines that MAAS manages.
+
+</WarningBox>
+
+- Each node in the PCG cluster requires a machine from MAAS in a ready state with the following resources:
+
+    <br />
+
+    - 4 CPUs
+    - 8192 MiB memory 
+    - 60 GiB storage
+
+<WarningBox>
+
+We recommend 100 GiB of storage for PCG nodes, as nodes can run out of 60 GoB of storage with prolonged use.
+
+</WarningBox>
```

Comment:
We can combine this with the first blue callout box and state our recommendation. We can also add it as a prerequisite bullet. That way, we reduce two callout boxes. Right now we have eight callout boxes, and we should use these sparingly.

Reply:
Revised prereqs and removed 2 callout boxes.